src/create/schema/index.ts

```diff
@@ -71,11 +71,12 @@ export type LazyMap = Map<ZodType, true>;
 export interface SchemaState {
   components: ComponentsObject;
   type: CreationType;
+  path?: string[];
```

Comment:
Given we are adding this with every iteration could we make both path and visited required?

Reply:
yeah that's probably best